CMakeLists.txt

```diff
@@ -1519,8 +1535,8 @@ if(WOLFSSL_EXAMPLES)
         tests/srp.c
         tests/suites.c
         tests/unit.c
-        examples/server/server.c
-        examples/client/client.c)
+        test_apps/server/test_server.c
```

Comment:
It should still build the original examples/server/server.c and examples/client/client.c

Reply:
I kept this change so that unit.test would pass